### PR TITLE
support settings in docker plugins install

### PR DESCRIPTION
**Pull request by vieux**
`docker plugin install tiborvass/no-remove DEBUG=1`

### CODE REVIEW

File: cli/command/plugin/install.go [hunk 1]
```diff
@@ -18,16 +18,20 @@ type pluginOptions struct {
...
 	grantPerms bool
 	disable    bool
+	args       []string
 }
 
 func newInstallCommand(dockerCli *command.DockerCli) *cobra.Command {
 	var options pluginOptions
 	cmd := &cobra.Command{
-		Use:   "install [OPTIONS] PLUGIN",
+		Use:   "install [OPTIONS] PLUGIN [key=value...]",
```
Comment (AkihiroSuda) [inline cli/command/plugin/install.go]: nit: `key=value` -> `KEY=VALUE` ?

Comment (vieux) [inline cli/command/plugin/install.go]: I even think it should be `[NAME[.FIELD][=VALUE]...]` and same in set. WDYT @tiborvass ?

Comment (thaJeztah) [inline cli/command/plugin/install.go]: In that case the "key" is `name.field`, so not sure we should complicate it further. We should probably explain it in other parts of the docs

**Comment by vieux** [comment]
@tiborvass PTAL

**Review by dnephin — APPROVED** [review]
LGTM

We don't do this "`key=value` as positional args" anywhere else, but I guess that's fine. It seems to work well here.

**Comment by anusha-ragunathan** [comment]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**